0.6.1 SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xd23b47]

while doing some testing 

sudo /root/headscale -n optim nodes list
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xd23b47]

goroutine 1 [running]:
github.com/juanfont/headscale/cmd/headscale/cli.nodesToPtables(0xc0003d6000, 0xb, 0x14, 0xc0000ab560, 0x0, 0x0, 0x0, 0xc0000aacf0)
	/home/runner/work/headscale/headscale/cmd/headscale/cli/nodes.go:167 +0x2c7
github.com/juanfont/headscale/cmd/headscale/cli.glob..func8(0x179eee0, 0xc0004da1e0, 0x0, 0x2)
	/home/runner/work/headscale/headscale/cmd/headscale/cli/nodes.go:92 +0x2a5
github.com/spf13/cobra.(*Command).execute(0x179eee0, 0xc0004da1a0, 0x2, 0x2, 0x179eee0, 0xc0004da1a0)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:856 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x179d5e0, 0x0, 0xc00031fd28, 0xd1e937)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
github.com/juanfont/headscale/cmd/headscale/cli.Execute()
	/home/runner/work/headscale/headscale/cmd/headscale/cli/root.go:24 +0x31
main.main()
	/home/runner/work/headscale/headscale/cmd/headscale/headscale.go:64 +0x287

well heres another... seems cannot list nodes in "plain text"

root@portaladmin:~# ./headscale nodes list -n optim 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xd27747]

goroutine 1 [running]:
github.com/juanfont/headscale/cmd/headscale/cli.nodesToPtables(0xc00059a000, 0x11, 0x14, 0xc000560c78, 0x0, 0x0, 0x0, 0xc0000b7488)
	/home/runner/work/headscale/headscale/cmd/headscale/cli/nodes.go:167 +0x2c7
github.com/juanfont/headscale/cmd/headscale/cli.glob..func8(0x17a5ee0, 0xc000544820, 0x0, 0x2)
	/home/runner/work/headscale/headscale/cmd/headscale/cli/nodes.go:92 +0x2a5
github.com/spf13/cobra.(*Command).execute(0x17a5ee0, 0xc0005447e0, 0x2, 0x2, 0x17a5ee0, 0xc0005447e0)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:856 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x17a45e0, 0x0, 0xc00029fd28, 0xd22417)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
github.com/juanfont/headscale/cmd/headscale/cli.Execute()
	/home/runner/work/headscale/headscale/cmd/headscale/cli/root.go:24 +0x31
main.main()
	/home/runner/work/headscale/headscale/cmd/headscale/headscale.go:64 +0x287

--- a/cmd/headscale/cli/nodes.go
+++ b/cmd/headscale/cli/nodes.go
@@ -154,8 +154,10 @@ func nodesToPtables(m []headscale.Machine) (pterm.TableData, error) {
 			ephemeral = true
 		}
 		var lastSeen time.Time
+		var lastSeenTime string
 		if m.LastSeen != nil {
 			lastSeen = *m.LastSeen
+			lastSeenTime = lastSeen.Format("2006-01-02 15:04:05")
 		}
 		nKey, err := wgkey.ParseHex(m.NodeKey)
 		if err != nil {
@@ -164,12 +166,12 @@ func nodesToPtables(m []headscale.Machine) (pterm.TableData, error) {
 		nodeKey := tailcfg.NodeKey(nKey)
 
 		var online string
-		if m.LastSeen.After(time.Now().Add(-5 * time.Minute)) { // TODO: Find a better way to reliably show if online
+		if lastSeen.After(time.Now().Add(-5 * time.Minute)) { // TODO: Find a better way to reliably show if online
 			online = pterm.LightGreen("true")
 		} else {
 			online = pterm.LightRed("false")
 		}
-		d = append(d, []string{strconv.FormatUint(m.ID, 10), m.Name, nodeKey.ShortString(), m.IPAddress, strconv.FormatBool(ephemeral), lastSeen.Format("2006-01-02 15:04:05"), online})
+		d = append(d, []string{strconv.FormatUint(m.ID, 10), m.Name, nodeKey.ShortString(), m.IPAddress, strconv.FormatBool(ephemeral), lastSeenTime, online})
 	}
 	return d, nil
 }

